ingestor/.chalice/config.json

```diff
@@ -20,14 +20,16 @@
           "iam_policy_file": "policy-newtrains.json"
         },
         "bb_store_station_status": {
-          "iam_policy_file": "policy-bluebikes-store.json"
+          "iam_policy_file": "policy-bluebikes-store.json",
+          "lambda_timeout": 90
```

Comment:
Were these failing with default timeout?

Reply:
Yep just slightly. Discovered it when trying to add tracing. They were hitting timeout every time

Reply:
Is there data missing?

Reply:
Doesn't look like it in s3, but hard to tell until we start using this data somewhere